Makefile

```diff
@@ -1,52 +1,28 @@
-# Shortcut targets
-default: test
+PACKAGE_NAME=github.com/projectcalico/libcalico-go
+GO_BUILD_VER=v0.27
 
-## Build binary
-all: test
+###############################################################################
+# Download and include Makefile.common before anything else
+###############################################################################
+MAKE_BRANCH?=$(GO_BUILD_VER)
+MAKE_REPO?=https://raw.githubusercontent.com/projectcalico/go-build/$(MAKE_BRANCH)
+WGET?=/usr/bin/wget
```

Comment:
@rafaelvanoni, apologies for not being clear earlier. I meant to suggest --

```
wget:
ifeq ($(shell which wget),)
    $(error "missing wget, please install wget to proceed")
endif

Makefile.common.$(MAKE_BRANCH): wget
```

Reply:
Yeah but that's overkill :-)

Reply:
Not to overemphasize but this could also be a place to lookup `curl` if `wget` is not present.